### ISSUE TITLE
TypeError after git init

### ISSUE THREAD

**Issue by eddiewang** [rangle/rangle-starter#124]
```
git clone git@github.com:rangle/react-redux-starter --depth=1 TinderMessenger
Cloning into 'TinderMessenger'...

git init
TypeError: Assignment to constant variable.
    at initRepo (/usr/local/lib/node_modules/rangle-starter/src/execute.js:50:13)
    at git.then.then.then (/usr/local/lib/node_modules/rangle-starter/src/execute.js:29:17)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

Does anyone know what's causing this issue?

**Comment by mhazy** [comment]
Which version of node were you using?

**Comment by eddiewang** [comment]
6.0.0 - FYI switching to 5.6.0 using n worked for me.

**Comment by mhazy** [comment]
It looks like this should be resolved once we release an updated package. Thanks for logging the issue, we'll get a release ready.

(Fixed in #104)

**Comment by rbrander** [comment]
I checked this today and it worked for me using node v6.10.2 (latest stable).  Seeing as how #104 has been merged, and I'm unable to reproduce this, I'm going to assume this is resolved and close this issue. 

@eddiewang If you are still having issues, please open a new issue.
